package.json

```diff
@@ -37,29 +38,37 @@
     "xmldom": "0.1.x"
   },
   "devDependencies": {
+    "@types/node": "^14.14.5",
+    "@typescript-eslint/eslint-plugin": "^4.6.0",
+    "@typescript-eslint/parser": "^4.6.0",
     "body-parser": "^1.17.1",
-    "eslint": "^6.5.1",
+    "eslint": "^6.8.0",
     "express": "^4.16.2",
     "mocha": "^6.0.2",
     "passport": "0.4.x",
     "request": "^2.83.0",
     "should": "*",
-    "sinon": "^7.2.7"
+    "sinon": "^7.2.7",
+    "typescript": "^4.0.5"
   },
   "engines": {
     "node": ">= 8"
   },
   "scripts": {
-    "test": "npm run lint && mocha",
-    "lint": "./node_modules/.bin/eslint lib",
-    "lint:fix": "./node_modules/.bin/eslint --fix lib"
+    "test": "npm run lint && npm run tsc && mocha",
+    "tsc": "tsc",
+    "lint": "eslint --ext .ts src || true",
+    "lint:fix": "eslint --ext .ts --fix src || true",
```

Comment:
I understand that the purpose of `|| true` here is to convert a linting "failure" exit code to a "success" exit code. Effectively, it turns linting errors into warnings, which is fine for now. I confirmed the test suite passes, but there are many linting errors converted to warnings.  I see that most errors might be automatically fixed with `eslint --fix`.